int64 id
string name
float64[] polygonCoordinates
int64 epochValidFrom
int64 epochValidTo
